### PR TITLE
SW-1180 split the updatescript to use params from a config file

### PR DESCRIPTION
**Pull request by irlaec**
Move the params of the update script to a separate file, in order to use the same update script in different plugins

### CODE REVIEW

**Review by JFSolution — APPROVED** [review]

**Review by JFSolution — CHANGES_REQUESTED** [review]
we need to remove the octoprint_mrbeam part completly from the update script, so it can be a clone in in the netconnectd plugin

**Review by JFSolution — CHANGES_REQUESTED** [review]
please fix the reformatting, as this is not the same with black, but the rest is looking good

File: octoprint_mrbeam/scripts/update_script.py [hunk 1]
```diff
@@ -404,30 +427,48 @@ def loadPluginTarget(archive, folder):
...
+        folder=folder,
+        plugin_extracted_path_folder=plugin_extracted_path_folder,
+        origin_file="scripts/update_script_config.py",
+        destination_file="update_script_config.py",
+    )
+
+    return zip_file_path
+
+
+def _copy_file_to_working_dir(
```
Comment (JFSolution) [inline octoprint_mrbeam/scripts/update_script.py:456]: I like, that you split this in a new function, but please add a Google Doc string

Comment (irlaec) [reply octoprint_mrbeam/scripts/update_script.py:456]: It's added :)

Comment (irlaec) [reply octoprint_mrbeam/scripts/update_script.py:456]: It's added

**Review by JFSolution — APPROVED** [review]

**Pull request merged**